From fe315cb039092839958e2b19a974e7aca2febdbf Mon Sep 17 00:00:00 2001
From: Supreeth Venkatesh <supreeth.venkatesh@amd.com>
Date: Thu, 5 Aug 2021 22:45:48 -0500
Subject: [PATCH 1/1] ARM:dts:aspeed: Add PSP Soft fuse gpios
Content-Type: text/plain; charset="us-ascii"
Content-Transfer-Encoding: 7bit

Adds PSP Soft fuse gpios for PSP notification when it soft fuses the
part and XTRIG5, XTRIG6 support for triggering cool reset.

Signed-off-by: Supreeth Venkatesh <supreeth.venkatesh@amd.com>
---
 arch/arm/boot/dts/aspeed-bmc-amd-onyx.dts   | 4 ++--
 arch/arm/boot/dts/aspeed-bmc-amd-quartz.dts | 4 ++--
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/arch/arm/boot/dts/aspeed-bmc-amd-onyx.dts b/arch/arm/boot/dts/aspeed-bmc-amd-onyx.dts
index f45c6a5ade3b..97902e8577b8 100644
--- a/arch/arm/boot/dts/aspeed-bmc-amd-onyx.dts
+++ b/arch/arm/boot/dts/aspeed-bmc-amd-onyx.dts
@@ -388,8 +388,8 @@
 	/*K0-K7*/	"","","","","","","","",
 	/*L0-L7*/	"","","","","","","","",
 	/*M0-M7*/	"","","","","","","","",
-	/*N0-N7*/	"","","","","","","","ASSERT_BMC_READY",
-	/*O0-O7*/	"","","HDT_SEL","","","JTAG_TRST_N","","",
+	/*N0-N7*/	"","","","","","","PSP_SOFT_FUSE_NOTIFY","ASSERT_BMC_READY",
+	/*O0-O7*/	"","","HDT_SEL","HDT_XTRIG5","HDT_XTRIG6","JTAG_TRST_N","","",
 	/*P0-P7*/	"MON_RST_BTN_L","ASSERT_RST_BTN_L","MON_PWR_BTN_L","ASSERT_PWR_BTN_L","","ASSERT_NMI_BTN_L","MON_PWR_GOOD","",
 	/*Q0-Q7*/	"","","HDT_DBREQ_L","","","","","",
 	/*R0-R7*/	"","","","","","","","",
diff --git a/arch/arm/boot/dts/aspeed-bmc-amd-quartz.dts b/arch/arm/boot/dts/aspeed-bmc-amd-quartz.dts
index a6beec4dd5c9..191afa81a9f1 100644
--- a/arch/arm/boot/dts/aspeed-bmc-amd-quartz.dts
+++ b/arch/arm/boot/dts/aspeed-bmc-amd-quartz.dts
@@ -294,8 +294,8 @@
 	/*K0-K7*/	"","","","","","","","",
 	/*L0-L7*/	"","","","","","","","",
 	/*M0-M7*/	"","","","","","","","",
-	/*N0-N7*/	"","","","","","","","ASSERT_BMC_READY",
-	/*O0-O7*/	"","","HDT_SEL","","","JTAG_TRST_N","","",
+	/*N0-N7*/	"","","","","","","PSP_SOFT_FUSE_NOTIFY","ASSERT_BMC_READY",
+	/*O0-O7*/	"","","HDT_SEL","HDT_XTRIG5","HDT_XTRIG6","JTAG_TRST_N","","",
 	/*P0-P7*/	"MON_RST_BTN_L","ASSERT_RST_BTN_L","MON_PWR_BTN_L","ASSERT_PWR_BTN_L","","ASSERT_NMI_BTN_L","MON_PWR_GOOD","",
 	/*Q0-Q7*/	"","","HDT_DBREQ_L","","","","","",
 	/*R0-R7*/	"","","","","","","","",
--
2.17.1

